substrate/client/chain-spec/Cargo.toml

```diff
@@ -16,6 +16,7 @@ workspace = true
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+clap = { version = "4.5.3", features = ["derive"] }
```

Comment:
one more thought: maybe clap shall be optional?
That would allow to use this module w/o all clap deps. If anyone needs to expose types to cli, the `clap` feature could be enabled.
 @gupnik wdyt?

Reply:
Yeah, makes sense. Will update it accordingly. Thanks.